lib/src/Uncertainty/Algorithm/Simulation/SubsetSampling.cxx

```diff
@@ -344,7 +344,7 @@ void SubsetSampling::initializeSeed(NumericalScalar threshold)
 NumericalScalar SubsetSampling::computeVarianceGamma(NumericalScalar currentFailureProbability, NumericalScalar threshold)
 {
   const UnsignedInteger N = currentPointSample_.getSize();
-  const UnsignedInteger Nc = std::max<UnsignedInteger>(1, targetProbability_ * N);
+  const UnsignedInteger Nc = std::max<UnsignedInteger>(1, conditionalProbability_ * N);
```

Comment:
The quantities targetProbability_ * N and conditionalProbability_ * N are used over and over. Maybe they should be computed once and for all.